templates/3c_generate_qc_metrics.ipynb

```diff
@@ -95,10 +95,12 @@
    "id": "12a31336",
```

Comment:
<p>Add general description of goals and expected output here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>